app/views/shared/_favicon.html.erb

```diff
@@ -1,20 +1,20 @@
-<link rel="apple-touch-icon-precomposed" sizes="57x57" href="/public/favicon/apple-touch-icon-57x57.png" />
-<link rel="apple-touch-icon-precomposed" sizes="114x114" href="/public/favicon/apple-touch-icon-114x114.png" />
-<link rel="apple-touch-icon-precomposed" sizes="72x72" href="/public/favicon/apple-touch-icon-72x72.png" />
-<link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/favicon/apple-touch-icon-144x144.png" />
-<link rel="apple-touch-icon-precomposed" sizes="60x60" href="/public/favicon/apple-touch-icon-60x60.png" />
-<link rel="apple-touch-icon-precomposed" sizes="120x120" href="/public/favicon/apple-touch-icon-120x120.png" />
-<link rel="apple-touch-icon-precomposed" sizes="76x76" href="/public/favicon/apple-touch-icon-76x76.png" />
-<link rel="apple-touch-icon-precomposed" sizes="152x152" href="/public/favicon/apple-touch-icon-152x152.png" />
-<link rel="icon" type="image/png" href="/public/favicon/favicon-196x196.png" sizes="196x196" />
-<link rel="icon" type="image/png" href="/public/favicon/favicon-96x96.png" sizes="96x96" />
-<link rel="icon" type="image/png" href="/public/favicon/favicon-32x32.png" sizes="32x32" />
-<link rel="icon" type="image/png" href="/public/favicon/favicon-16x16.png" sizes="16x16" />
-<link rel="icon" type="image/png" href="/public/favicon/favicon-128.png" sizes="128x128" />
+<link rel="apple-touch-icon-precomposed" sizes="57x57" href="<%=image_path('favicon/apple-touch-icon-57x57.png')%>" />
```

Comment:
You can also use https://apidock.com/rails/ActionView/Helpers/AssetTagHelper/favicon_link_tag - with asset path - but if this all works, then *great* !